driver_test.go

```diff
@@ -1263,6 +1264,48 @@ func testArray(t *testing.T, json bool) {
 	})
 }
 
+func TestVector(t *testing.T) {
```

Comment:
Do we need to enable some parameter for this to work?

Reply:
I wasn't sure which environment these tests were running oi. However, it looks like the CI failures are due to the parameter, so I've added a statement to enable it for this test.